core/node/node_sync/src/tree_data_fetcher/provider/mod.rs

```diff
@@ -89,6 +94,14 @@ struct PastL1BatchInfo {
     number: L1BatchNumber,
     l1_commit_block_number: U64,
     l1_commit_block_timestamp: U256,
+    chain_id: SLChainId,
+}
+
+#[derive(Debug)]
+struct SLChainData {
```

Comment:
Bikeshedding: IMO, `Data` suffix is not the best option given that its key part is a client. Maybe, `SLChainClient` / `SLChainAccess`?

Reply:
agree, `SLChainAccess` sounds better